gc/base/CollectorLanguageInterface.cpp

```diff
@@ -36,14 +36,26 @@ MM_CollectorLanguageInterface::writeBarrierStore(OMR_VMThread *omrThread, omrobj
 
 #if defined(OMR_GC_MODRON_CONCURRENT_MARK) || defined(OMR_GC_MODRON_SCAVENGER)
 	MM_EnvironmentStandard *env = MM_EnvironmentStandard::getEnvironment(omrThread);
-	MM_GCExtensionsBase *extensions = env->getExtensions();
+	generationalWriteBarrierStore(env, parentObject, childObject);

```

Comment:
Can this code be broken up so that the code the scavenger and the concurrent code are not in the same #if block. It would make the code clearer as to what was happening.  Similarly for the writeBarrierUpdate below.